### PR TITLE
feat(route): Add 福利吧最近更新内容

### PR DESCRIPTION
**Pull request by shinemoon**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/fuliba/latest
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [x] CN / 中文文档
- [x] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/fuliba/latest.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const domain = ctx.query.domain ?? 'fuliba.net';
+    const res = await got(`https://${domain}`, {
```
Comment (TonyRL) [inline lib/v2/fuliba/latest.js:6]: GHSA-64wp-jh9p-5cg2

Comment (shinemoon) [reply lib/v2/fuliba/latest.js:6]: Any action I need to take, or? :)

Comment (TonyRL) [reply lib/v2/fuliba/latest.js:6]: This introduce an obvious vector for SSRF attacks. Either remove it or it won't be merged.

Comment (shinemoon) [reply lib/v2/fuliba/latest.js:6]: Got it, thanks. Then I will refine/re-submit there. 

> This introduce an obvious vector for SSRF attacks. Either remove it or it won't be merged.

File: lib/v2/fuliba/radar.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const fuliba = {
+    _name: '福利吧',
+    '.': [
+        {
+            title: '福利吧',
```
Comment (TonyRL) [inline lib/v2/fuliba/radar.js:5]:
```suggestion
            title: '最新',
```

File: website/docs/routes/new-media.mdx [hunk 3]
```diff
@@ -743,6 +743,9 @@ Country
 
 <Route author="nczitzk" example="/foresightnews/column/1" path="/foresightnews/column/:id" paramsDesc={['专栏 id, 可在对应专栏页 URL 中找到']}/>
 
+
+
+
```
Comment (TonyRL) [inline website/docs/routes/new-media.mdx:748]:
```suggestion
```
Unnessary addition

File: website/docs/routes/new-media.mdx [hunk 4]
```diff
@@ -3293,6 +3296,14 @@ Type 栏目：
...
 </Route>
 
+## 福利吧 {#fu-li-ba}
+
+### 最新 {#fu-li-ba-zui-xin}
+
+<Route author="shinemoon" example="/fuliba/latest" path="/fuliba/latest"/>
+
+
+
```
Comment (TonyRL) [inline website/docs/routes/new-media.mdx:3306]:
```suggestion
<Route author="shinemoon" example="/fuliba/latest" path="/fuliba/latest"/>

```
One linebreak is enough

File: lib/v2/fuliba/latest.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        });
+
+    const resultItems = await Promise.all(
+        count.map((i) =>
+            ctx.cache.tryGet(i.link, async () => {
+                i.description = await fetchContent(i.link);
+                return i;
+            })
+        )
+    );
```
Comment (TonyRL) [inline lib/v2/fuliba/latest.js:33]: Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

You can list recent posts using `/wp-json/wp/v2/posts`. If you want categories and tags for each post in the posts API, you can add the `_embed` search parameter.

To adjust the no. of articles returned in the posts API, you can pass the `per_page` search parameter.

Comment (shinemoon) [reply lib/v2/fuliba/latest.js:33]: OK, honestly I even didn't realize that site is built on top of WP....interesting.

I need to take some time to refine this part , maybe update tomorrow. thanks.

**Pull request merged**